components/lib/dropdown/DropdownPanel.js

```diff
@@ -144,17 +113,14 @@ export const DropdownPanel = React.memo(
                 );
 
                 return (
-                    <React.Fragment key={key}>
-                        <li {...itemGroupProps}>
+                    <li key={key} {...itemGroupProps}>
                             <span {...itemGroupLabelProps}>{groupContent}</span>
-                        </li>
-                        {groupChildrenContent}
-                    </React.Fragment>
-                );
+                    </li>
+                )
+                
             }
-
-            const optionLabel = props.getOptionLabel(option);
             const optionKey = index + '_' + props.getOptionRenderKey(option);
```

Comment:
Group Item key not the same as it was, i do not use here group index to create render key, is this okey ?

Reply:
Same thing, but the index is at the back